fluid-cpp/fluid_solver.cpp

```diff
@@ -0,0 +1,173 @@
+#include "utilities.h"
+#include "FluidComputeSolution.h"
+
+#include "precice/SolverInterface.hpp"
+#include <iostream>
+#include <mpi.h>
+#include <vector>
+
+using namespace precice;
+using namespace precice::constants;
+
+int main(int argc, char** argv)
+{
+
+  std::cout << "Starting Fluid Solver..." << std::endl;
+  if (argc != 5 && argc != 6) {
+    std::cout << std::endl;
+    std::cout << "Fluid: Usage: mpiexec -np <#procs> " << argv[0] << " <configurationFileName> <N> <tau> <kappa> -parallel" << std::endl;
```

Comment:
Since this is always tedious to lookup, let's add simple `run.sh` scripts, similarly to the tutorials. Here they will probably not be longer than five lines.